.promu.yml

```diff
@@ -12,7 +12,7 @@ build:
           path: ./cmd/promtool
         - name: tsdb
           path: ./tsdb/cmd/tsdb
-    flags: -mod=vendor -a -tags netgo
+    flags: -mod=vendor -a -tags netgo,builtinassets
```

Comment:
this creates
> cmd/go: -tags space-separated list contains comma
when running make build

Reply:
You'll need Go 1.13 as specified in this file as well, see https://github.com/prometheus/prometheus/pull/6308

Reply:
Thx